src/System.Drawing.Common/tests/IconTests.cs

```diff
@@ -385,7 +385,25 @@ public void XpIcon_ToBitmap_Success(int size)
         [ConditionalFact(Helpers.IsDrawingSupported)]
         public void ExtractAssociatedIcon_FilePath_Success()
         {
-            using (Icon icon = Icon.ExtractAssociatedIcon(Helpers.GetTestBitmapPath("48x48_multiple_entries_4bit.ico")))
+            ExtractAssociatedIcon_FilePath_Success(Helpers.GetTestBitmapPath("48x48_multiple_entries_4bit.ico"));
+        }
+
+        [ActiveIssue(20884, TestPlatforms.AnyUnix)]
+        [ConditionalFact(Helpers.IsDrawingSupported)]
+        public void ExtractAssociatedIcon_UNCFilePath_Success()
+        {
+            string bitmapPath = Helpers.GetTestBitmapPath("48x48_multiple_entries_4bit.ico");
+            string bitmapPathRoot = Path.GetPathRoot(bitmapPath);
+            string bitmapUncPath = $"\\\\{Environment.MachineName}\\{bitmapPath.Substring(0, bitmapPathRoot.IndexOf(":"))}$\\{bitmapPath.Replace(bitmapPathRoot, "")}";
```

Comment:
I have no idea whether this test is going to be stable (eg., it is possible that the administrative paths `\\localhost\x$` are not accessible in some machine)
It might be best to check whether they can be accessed first, and if not, just pass the test without trying to load the icon.

Reply:
done thank's!